core/chains/evm/gas/rollups/l1_oracle.go

```diff
@@ -33,6 +32,11 @@ type l1OracleClient interface {
 	BatchCallContext(ctx context.Context, b []rpc.BatchElem) error
 }
 
+type EVMClient interface {
```

Comment:
We already know that we are in the `evm` subtree, so would this still be clear as just `Client`?
```suggestion
type Client interface {
```